### PR TITLE
Updated package.json for v4 API

### LINKED ISSUE #191: Upgrade to Ghost API v4
Recently Ghost upgraded to v4, so we should change the API (see `package.json`) to v4 too.

### PR DESCRIPTION
**Pull request by faisalsgithub**
Closes #191

### CODE REVIEW

File: assets/js/eventCardFetcher.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ $(function () {
     var api = new GhostContentAPI({
         url: "https://beta.advisory.sg",
         key: "336313a0318904e945f0938d06",
-        version: "v3",
+        version: "v4",
```
Comment (wei2912) [inline assets/js/eventCardFetcher.js:5]: Were there any API changes from v3 to v4 that might affect this?

**Review by wei2912 — COMMENTED** [review]
Not sure if this PR is complete - see comments. Thanks!

File: assets/js/eventCardFetcher.js (hunk 1 above)
Comment (bnjmnt4n) [reply assets/js/eventCardFetcher.js:5]: There doesn't seem to be any related to the content API: https://ghost.org/docs/changes/#ghost-40.

File: package.json [hunk 2]
```diff
@@ -2,10 +2,9 @@
     "name": "ghost-advisory-theme",
     "description": "Custom-built Ghost theme for Advisory SG",
     "demo": "https://advisory.sg",
-    "version": "1.0.0",
+    "version": "4.4.0",
```
Comment (bnjmnt4n) [inline package.json:5]: It's probably not necessary to update this version: it refers to the version of our theme package, not the version of Ghost. As a side note, maybe we might want to change it to 0.0.0 or 1.0.0-beta to indicate that it's not really done/production-ready.

**Comment by bnjmnt4n** [comment]
More relevant for the v4 upgrade might be:

> Themes: @site.lang has been deprecated in favour of @site.locale.

if we want to consolidate the changes into 1 PR.

File: package.json (hunk 2 above)
Comment (wei2912) [reply package.json:5]: I think let's change to 0.0.0.

**Review by wei2912 — APPROVED** [review]
LGTM!

**Pull request merged**